client/src/home/edit_form/EditForm.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from "react";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import DatePicker from "react-datepicker";
+import NumericInput from "react-numeric-input";
+import { useCookies } from "react-cookie";
+import { editItem } from "../../MongodbFunctions";
+import { ReactSearchAutocomplete } from 'react-search-autocomplete';
+
+import "react-datepicker/dist/react-datepicker.css";
+import "./EditForm.scss";
+
+function EditTempButton() {
+    const [formRendered, toggleFormRendered] = useState(false);
+  
+    return (
+      <div className="EditForm">
+        <Button variant="dark" onClick={() => toggleFormRendered(!formRendered)}>
+          Edit Item
+        </Button>
+        {formRendered && (
+          <EditForm closeForm={() => toggleFormRendered(false)}></EditForm>
+        )}
+      </div>
+    );
+  }
+  
+  function EditForm(props) {
+    // Input fields
+    const [itemName, setItemName] = useState("");
+    const [oldBatchId, setOldBatchId] = useState("");
+    const [newBatchId, setNewBatchId] = useState("");
+    const [lbPerHouse, setLbPerHouse] = useState(0);
+    const [weight, setWeight] = useState(0);
+    const [outByDate, setOutByDate] = useState(new Date());
+    
+    const [cookies] = useCookies(["token"]);
+    const [showWeightWarning, setWeightWarning] = useState(false);
+    const [showitemNameWarning, setitemNameWarning] = useState(false);
+    const [showbatchIdWarning, setbatchIdWarning] = useState(false);
+    // Logic for adding recommendations will need to be added
+    const [rec, setRec] = useState([]);
+    const [itemList, setItemList] = useState([]);
+  
+    useEffect(() => {
```

Comment:
I'm a little confused, in your EditTempButton, it doesn't look like you pass any itemList in as a prop, so where is this coming from and why do you need it?